### ISSUE TITLE
What is semantic HTML?

### ISSUE THREAD

**Issue by erikmontes** [h5bp/Front-end-Developer-Interview-Questions#208]
What's the importance and benefits?

**Comment by darcyclarke** [comment]
Not sure I understand your question. Could you rephrase it? Is this supposed to be a question you'd like added to the list?

**Comment by tarzak** [comment]
Readability in maintenance and possibility to "read" web page by voice systems properly?

**Comment by erikmontes** [comment]
@darcyclarke Yes, if it seems like a worthy question.

@tarzak also for SEO

**Comment by jecel0911** [comment]
maybe, why is important semantic HTML?

A/ SEO, readability of the code.

**Comment by darcyclarke** [comment]
@erikmontes Feel free to add it as a Pull Request. Make sure to squash your commits and leave out any answers.
